nova_vm/src/ecmascript/builtins/control_abstraction_objects/promise_objects/promise_abstract_operations.rs

```diff
@@ -2,7 +2,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-pub(crate) mod promise_capability_records;
-pub(crate) mod promise_jobs;
-pub(crate) mod promise_reaction_records;
-pub(crate) mod promise_resolving_functions;
+pub mod promise_capability_records;
+pub mod promise_jobs;
```

Comment:
**question:** Are the promise_jobs, promise_constructor, promise_prototype, promise_abstract_operations necessary to expose?

Reply:
It seems like I exposed too much, just pushed a fix, the rest is 100% needed.